app/client.js

```diff
@@ -50,13 +51,22 @@ const piwikPlugin = {
   plugContext: plugContext(addPiwik),
 };
 
+const Raven = getRaven(config);
 const addRaven = context => (context.raven = Raven); // eslint-disable-line no-param-reassign
 
 const ravenPlugin = {
   name: 'RavenPlugin',
   plugContext: plugContext(addRaven),
 };
 
+const addConfig = context =>
+ (context.config = window.config); // eslint-disable-line no-param-reassign
+
+const configPlugin = {
```

Comment:
In the current setup, this makes the config available via fluxible context. Removewthe plugin -> code stops working. Maybe the same functionality can be achieved in some other (simpler?)  way.